src/components/sbb-option/sbb-option.scss

```diff
@@ -0,0 +1,88 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  @include sbb.text-s--regular;
```

Comment:
Is this really necessary? we normally try to avoid styles on the host because it's not encapsulated anymore